opencensus/tags/tag_map.py

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from collections import OrderedDict
```

Comment:
`tag_map` doesn't need to be ordered.

Reply:
`dict` broke `test_to_byte_array` which relied on specific order.
though from python 3.6 dict is ordered by design.
https://stackoverflow.com/questions/39980323/are-dictionaries-ordered-in-python-3-6

thoughts?

Reply:
I see, then I think it's fine to leave it as a OrderedDict to be consistent with the language spec.